__tests__/utilityFunctions.test.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const fileData = JSON.parse(fs.readFileSync('./user.json'));
+const { validationUserPost, addUserPost, putUserbyId, deleteUserById, getUserById, readFromUserFile } = require("../utilityFunctions");
+
+/*
+the function below i made because i thought it is maybe better to don't use hardcoded data, if it is better like this then the user and objects can be replaced by  
+randomWholeUserFromFile and randomUserNameFromFile, if not i will remove this in the next PR.
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
+const userNumber = getRandomInt(fileData.length);
+const randomWholeUserFromFile = fileData[userNumber];
+const randomUserNameFromFile = randomWholeUserFromFile.name;
+*/
+
+//this test is not perfect because i still have this issue with the append part mentioned in the utilityFunctions file.
+test('testPostAddUser', () => {
+  expect(addUserPost({
```

Comment:
It's ok this way, but maybe it's clearer if you make the call, store the result in a variable, and then check for the results.

Doing everything in a single like might make it harder to read, specially in more complex calls.